conductor-client/src/main/kotlin/com/openlattice/assembler/AssemblerConnectionManager.kt

```diff
@@ -115,11 +112,91 @@ class AssemblerConnectionManager(
             createSchema(dataSource, TRANSPORTER_SCHEMA)
             createSchema(dataSource, ASSEMBLED_ENTITY_SETS)
             configureOrganizationUser(organizationId, dataSource)
-            addMembersToOrganization(organizationId, dataSource, organizations.getMembers(organizationId))
+            addMembersToOrganization(organizationId, dataSource, securePrincipalsManager.getOrganizationMemberPrincipals(organizationId))
             configureServerUser(dataSource)
         }
     }
 
+    fun createAndInitializeCollaborationDatabase(collaborationId: UUID, dbName: String): Int {
```

Comment:
Would prefer moving collaboration specific calls into their own place, maybe in the Collaboration service. Alternatively would prefer replacing 'functionality*' calls with generic calls located in a more generic place.
EX:
- Assembler calling ExternalDatabaseManagementService.createDatabase(<assembler specific things>)
- CollaborationService calling ExternalDatabaseManagementService.createDatabase(<collaboration specific things>)
VS 
- Assembler calling AssemblerConnectionManager.createOrganizationDatabase()
- CollaborationService calling AssemblerConnectionManager.createAndInitializeCollaborationDatabase()

Reply:
Just code organization so feel free to punt to a follow-up